charter.md

```diff
@@ -22,14 +22,20 @@ Server-initiated services are out of scope. The working group will first deliver
 a requirements document for IP proxying. Once both are complete, the working group will focus on a protocol solution for
 IP proxying.
 
-The working group will consider fallback to versions of HTTPS that operate over TCP as a mitigation to UDP or HTTP/3
-blocking. Moreover, the working group will consider implications of tunneling protocols with congestion control and loss
-recovery over MASQUE, and may issue recommendations accordingly. New congestion control and loss recovery algorithms are
+Exercising the extension points defined by CONNECT-UDP and CONNECT-IP is important to ensure their continued useability.
+The working group will consider extensions to these documents that provide missing functionality or enhance performance
+for real world use cases. The initial set of extensions will be in support of UDP listening, QUIC-aware proxying, and
```

Comment:
```suggestion
Exercising the extension points defined by CONNECT-UDP and CONNECT-IP helps make it easier to
support new use cases or accommodate changes in the environment in which these protocols are deployed.
As such, the working group will consider extensions to these documents that provide missing
functionality or otherwise ease deployability for real world use cases.
The initial set of extensions will be in support of UDP listening, QUIC-aware proxying, and
```

Reply:
Also folded in the later discussion, this is much nicer, thank you!